### PR TITLE
Add Selma to Montgomery National Historic Trail shield

### PR DESCRIPTION
**Pull request by quincylvania**
This one is a little different: since the historic trail consists pretty much of extant roads, there's no "auto tour route" separate from the trail.

https://www.openstreetmap.org/relation/13302056

https://commons.wikimedia.org/wiki/File:45th_Anniversary_of_the_Civil_Rights_March_from_Selma,_Alabama_to_Montgomery,_Alabama_LCCN2010637779.tif

Re: #1101

Screenshot 2024-06-30 at 11 38 17 AM

### CODE REVIEW

File: icons/shield_us_nht_semo.svg
Comment (1ec5) [inline icons/shield_us_nht_semo.svg:1]: Sky blue surrounded by white doesn’t have very much contrast against an already light background. Should we surround all that with another black outline, or would that be too heavy?

Comment (quincylvania) [reply icons/shield_us_nht_semo.svg:1]: I think this does look better, though is the shield getting too large now? We should probably have consistency across all the National Historic Trail shields (and look forward to National Scenic Trail shields, which are similar in design).

Screenshot 2024-07-01 at 10 28 11 AM

Comment (quincylvania) [reply icons/shield_us_nht_semo.svg:1]: Here is 22x21. The above is 24x23.

Screenshot 2024-07-01 at 11 07 40 AM

Comment (1ec5) [reply icons/shield_us_nht_semo.svg:1]: This gives the shield more definition, which is great.

As an aside, maybe we should consider putting an extra outline around anything we’re currently outlining in white, including the Interstate shield. Apple Maps takes this approach; sometimes it makes the shields look more cartoonish, but the added contrast really comes in handy over light backgrounds.

**Review by ZeLonewolf — APPROVED** [review]
Looks good to me!
image

**Pull request merged**